src/sbin/task_manager.pl.in

```diff
@@ -436,9 +436,9 @@ sub create {
     } else {
         $object = '_global';
         $task_file =
-              $spool_task . '/'
-            . $date . '.'
-            . $label . '.'
+              $spool_task . '/' 
+            . $date . '.' 
+            . $label . '.' 
```

Comment:
This hunk introduces trailing white spaces.

Reply:
Maybe a bug of perltidy. I fixed. Thanks.